integration/integration_test.go

```diff
@@ -63,6 +63,7 @@ func Test(t *testing.T) {
 	images = append(images, image{"containous/maesh:latest", false})
 	images = append(images, image{"containous/whoami:v1.0.1", true})
 	images = append(images, image{"containous/whoamitcp", true})
+	images = append(images, image{"containous/whoamiudp:dev", true})
```

Comment:
is `dev` the right tag to be using here?

Reply:
The `latest` tag does not exist for the [containous/whoamiudp](https://hub.docker.com/r/containous/whoamiudp/tags) image.

Reply:
Would be neat to push one

Reply:
After thinking a bit more about that, I've used the `digest` tag for the `whoamitcp` and `whoamiudp` images to make sure that the build is reproducible. We'll have to update the image tags when a version tags will be available.